src/SFML/Graphics/RenderTextureImplFBO.cpp

```diff
@@ -141,6 +321,18 @@ bool RenderTextureImplFBO::activate(bool active)
 ////////////////////////////////////////////////////////////
 void RenderTextureImplFBO::updateTexture(unsigned int)
 {
+
+#ifndef SFML_OPENGL_ES
+
+    if (m_multisampleFrameBuffer)
+    {
+        glCheck(GLEXT_glBindFramebuffer(GLEXT_GL_DRAW_FRAMEBUFFER, m_frameBuffer));
+        glCheck(GLEXT_glBlitFramebuffer(0, 0, m_width, m_height, 0, 0, m_width, m_height, GL_COLOR_BUFFER_BIT, GL_NEAREST));
```

Comment:
Wouldn't GL_LINEAR give smoother results? Or is the parameter irrelevant, because the textures have the same size?

Reply:
In the odd case the driver is stupid, this is faster and imposes no loss of quality.